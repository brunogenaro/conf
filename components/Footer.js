import React, { PureComponent } from 'react';
import Globals from '../utils/Globals';

import Box from './Box';

export default class extends PureComponent {
  render(){
    return(
      <div className='footer'>
        <Box>
          <h3>DATA</h3>
          <h2>{Globals.date.full}</h2>
          <p>10:00am - 17:00pm</p>
        </Box>
        <Box>
          <h3>LOCAL</h3>
          <h2>{Globals.location.city}</h2>
          <p>{Globals.location.locale}</p>
        </Box>

        <style jsx>{`
          .footer {
            align-self: flex-end;
            display: flex;
            @media(max-width: 720px) {
              align-self: auto;
            }
          }

          h3, p {
            color: ${Globals.colors.primary};
            margin: 0px;
            letter-spacing: 2px;
          }

          p {
            color: ${Globals.colors.primary};
            letter-spacing: 2px;
          }

          h2 {
            margin: 0px;
            letter-spacing: 6px;
          }
        `}
        </style>
      </div>
    );
  }
}