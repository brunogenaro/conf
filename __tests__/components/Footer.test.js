import React from 'react';
import renderer from 'react-test-renderer'
import Footer from '../../components/Footer.js';

describe('With Snapshot Testing', () => {
  it('Footer Snapshot Testing', () => {
    const component = renderer.create(<Footer />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot()
  })
});