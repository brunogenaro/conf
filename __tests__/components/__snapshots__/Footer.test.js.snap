// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Footer Snapshot Testing 1`] = `
<div
  className="footer"
  data-jsx={101808419}
>
  <div
    className="cont"
    data-jsx={3134665551}
  >
    <h3
      data-jsx={101808419}
    >
      DATA
    </h3>
    <h2
      data-jsx={101808419}
    >
      05.10.2017
    </h2>
    <p
      data-jsx={101808419}
    >
      10:00am - 17:00pm
    </p>
  </div>
  <div
    className="cont"
    data-jsx={3134665551}
  >
    <h3
      data-jsx={101808419}
    >
      LOCAL
    </h3>
    <h2
      data-jsx={101808419}
    >
      São Paulo
    </h2>
    <p
      data-jsx={101808419}
    >
      SENAI - Santa cecília 
    </p>
  </div>
</div>
`;
